### DISCUSSION TITLE
Build error

### DISCUSSION THREAD

**Discussion by Zhang-jie-jun** [ceph/go-ceph#538] [Q&A]
Has anyone encountered this problem before?

![image](https://user-images.githubusercontent.com/82801500/126587005-f324af01-3c9b-4ad1-b379-979d832ebc8e.png)

# go build test
# github.com/ceph/go-ceph/rados
/home/work/pkg/mod/github.com/ceph/go-ceph@v0.10.0/rados/ioctx.go:678:9: could not determine kind of name for C.rados_ioctx_get_namespace

**Comment by nixpanic** [comment]
What version of the Ceph libraries do you have installed? I guess the default include file `rados/librados.h` does not have `rados_ioctx_get_namespace()`. Current versions go-ceph work with Ceph Nautilus and newer.

**Comment by elrondwong** [comment]
I have the same problems.
ceph version: Nautilus
go-ceph version: v0.10.0

resolve it:
1 Update your Ceph libraries to Octopus, you can remote debug with dlv.
2 Use Nautilus, build with `go build -tags nautilus`, but you can't remote debug with dlv.
